src/pal/src/include/pal/dbgmsg.h

```diff
@@ -416,7 +361,6 @@ bool DBG_ShouldCheckStackAlignment();
     }                                                                   \
     if(g_Dbg_asserts_enabled)                                           \
     {                                                                   \
-        PAL_Leave();                                                    \
```

Comment:
Note: removed this call to match what the `__GNUC__` block was doing.  Got an error when this is left in.